### ISSUE TITLE
Spawn protection won't return player placed item

### ISSUE THREAD

**Issue by UnluckyNinja** [Pryaxis/TShock#1471]
Version: TShock Mintaka 4.3.24 for Terraria 1.3.5.3 (APIv2.1, OTAPI v2.0.0.27)

I authed in using the temporary token, but didn't promote myself to the admin group. Then I used /protectspawn to open the spawn. Played a bit, saved and quit.
The second day, I noticed that spawn was in protection again. The piggy bank I placed was removed but wasn't given back to me.
So I think it should be a bug.

Sorry for bothering if it's an intended behavior.

**Comment by tlworks** [comment]
Any logs, did you save your world? What exactly do you mean with 'Second day', in-game or real life time days?

**Comment by Enerdy** [comment]
Commands that modify config values directly from the game do not persist between sessions. If you want to permanently turn off spawn protection, you're going to have to change it in the `config.json` file inside `/tshock` and then use the `reload` command in the console.

**Comment by UnluckyNinja** [comment]
@tlworks Real life time. I think I wrote it in the post. But it's not the point. I'm just telling how I encountered that problem(the title one), an unexpected behavior, which missing a function that return placed item to the player, who is not allow to build in spawn.

@Enerdy Thanks, so this is why it's in protection agian! But is it that the protection won't return placed item to player, an intended behavior?
